src/Sync/Singular.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace WPML\ElasticPress\Sync;
+
+use ElasticPress\Indexables;
+
+use WPML\ElasticPress\Constants;
+use WPML\ElasticPress\Manager\Indices;
+
+use WPML\ElasticPress\Traits\CrudPropagation;
+
+class Singular {
+
+	use CrudPropagation;
+
+	/** @var Indexables */
+	private $indexables;
+
+	/** @var Indices */
+	private $indicesManager;
+
+	/** @var array */
+	private $activeLanguages;
+
+	/** @var string */
+	private $defaultLanguage;
+
+	/**
+	 * @param Indices    $indicesManager
+	 * @param Indexables $indexables
+	 * @param array      $activeLanguages
+	 * @param string     $defaultLanguage
+	 */
+	public function __construct(
+		Indexables $indexables,
+		Indices    $indicesManager,
```

Comment:
Indices and Indexables are exchanged, which one is correct @decodekult ?

Reply:
+1